docs/home/introduction.mdx

```diff
@@ -31,4 +20,47 @@ Open any third-party website or application, synchronize audio and video flawles
   ></video>
 </Example>
 
-Still have questions? Contact us at [founders@hyperbeam.com](mailto:founders@hyperbeam.com)
+### Run various applications:
+
+- Chromium (virtual browser)
+- Android emulator (<a href="https://docs.google.com/forms/d/e/1FAIpQLSdsVFZ4cQ4Q2nHNRhwb6SvmJIekv7fuQVa27zWnk_r5F48B5w/viewform?usp=pp_url&entry.126298931=Android+emulation" target="_blank" rel="noopener noreferrer">Request access</a>)
+- NES emulator (<a href="https://docs.google.com/forms/d/e/1FAIpQLSdsVFZ4cQ4Q2nHNRhwb6SvmJIekv7fuQVa27zWnk_r5F48B5w/viewform?usp=pp_url" target="_blank" rel="noopener noreferrer">Request access</a>)
+- Want to run something else? Let us know at [founders@hyperbeam.com](mailto:founders@hyperbeam.com) or <a href="https://docs.google.com/forms/d/e/1FAIpQLSdsVFZ4cQ4Q2nHNRhwb6SvmJIekv7fuQVa27zWnk_r5F48B5w/viewform?usp=pp_url" target="_blank" rel="noopener noreferrer">request it here</a>
+
+
+### Supported 3D JS Libraries
+
+<CardGroup cols={2}>
+  <Card title="Three.js Example" icon={<img src="https://mintlify.s3-us-west-1.amazonaws.com/hyperbeam/assets/threejs-logo.svg"/>} color="#a888ff" href="https://github.com/hyperbeam/threejs-example">
+  </Card>
+  <Card title="A-Frame Example" icon={<img src="https://mintlify.s3-us-west-1.amazonaws.com/hyperbeam/assets/aframe-logo.png"/>} color="#a888ff" href="https://github.com/hyperbeam/aframe-example">
+  </Card>
+  <Card title="React Three Fiber Example" icon={<img src="https://mintlify.s3-us-west-1.amazonaws.com/hyperbeam/assets/react-logo.svg"/>} color="#a888ff" href="https://github.com/hyperbeam/react-three-fiber-example">
+  </Card>
+  <Card title="Babylon.js Example" icon={<img src="https://mintlify.s3-us-west-1.amazonaws.com/hyperbeam/assets/babylonjs-logo.svg"/>} color="#a888ff" href="https://github.com/hyperbeam/babylonjs-example">
+  </Card>
+</CardGroup>
+
+### Client SDKs
+
+<CardGroup cols={3}>
+  <Card title="JavaScript SDK" icon="js" color="#a888ff" href="/client-sdk/javascript">
+	</Card>
+  <Card title="Unity SDK (WebGL)" icon="unity" color="#a888ff" href="/client-sdk/unity/unity-webgl-sdk">
+	</Card>
+  <Card title="Unity SDK (Desktop)" icon="unity" color="#a888ff" href="/client-sdk/unity/unity-desktop-sdk">
+	</Card>
+</CardGroup>
+
+### Other Resources
+
+<CardGroup cols={2}>
+  <Card title="REST API" icon="window" color="#a888ff" href="/rest-api/dispatch">
+  	Create and manage virtual computer sessions server-side.
```

Comment:
Seems we're mixing tabs and spaces here. Let's add something like prettier (see how the example repo does it) and lint in another commit